imap_processing/glows/packet_definitions/GLX_COMBINED.xml

```diff
@@ -0,0 +1,209 @@
+			</xtce:Parameter>
+			<xtce:Parameter name="OFFSETSEC" parameterTypeRef="UINT16">
+				<xtce:LongDescription>Block end time (IMAP), seconds</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="OFFSETSUBSEC" parameterTypeRef="UINT24">
+				<xtce:LongDescription>Block end time (IMAP), subseconds</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="GLXSEC" parameterTypeRef="INT32">
+				<xtce:LongDescription>Block start time (GLOWS), seconds</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="GLXSUBSEC" parameterTypeRef="UINT24">
+				<xtce:LongDescription>Block start time (GLOWS), Subseconds</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="GLXOFFSEC" parameterTypeRef="UINT16">
+				<xtce:LongDescription>Block end time (GLOWS), seconds</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="GLXOFFSUBSEC" parameterTypeRef="UINT24">
+				<xtce:LongDescription>Block end time (GLOWS), subseconds</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SPINS" parameterTypeRef="UINT8">
+				<xtce:LongDescription>Number of spins</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="NBINS" parameterTypeRef="UINT16">
+				<xtce:LongDescription>Number of bins</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="TEMPAVG" parameterTypeRef="UINT8">
+				<xtce:LongDescription>Mean filter temperature </xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="TEMPVAR" parameterTypeRef="UINT16">
+				<xtce:LongDescription>Variance of filter temperature </xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="HVAVG" parameterTypeRef="UINT16">
+				<xtce:LongDescription>Mean CEM voltage</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="HVVAR" parameterTypeRef="INT32">
+				<xtce:LongDescription>Variance of CEM voltage </xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SPAVG" parameterTypeRef="UINT16">
+				<xtce:LongDescription>Mean spin period </xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SPVAR" parameterTypeRef="INT32">
+				<xtce:LongDescription>Variance of spin period </xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="ELAVG" parameterTypeRef="UINT8">
+				<xtce:LongDescription>Mean length of event impulse</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="ELVAR" parameterTypeRef="UINT16">
+				<xtce:LongDescription>Variance of event-impulse length</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="EVENTS" parameterTypeRef="UINT32">
+				<xtce:LongDescription>Number of events</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="HISTOGRAM_DATA" parameterTypeRef="BYTE28800">
+				<xtce:LongDescription>Histogram Counts</xtce:LongDescription>
+			</xtce:Parameter>
+			<!--Direct events sequence containers-->
```

Comment:
There is a name collision in the histogram and direct events packets - they both have a parameter named `SEC`. I could update the name (`DE_SEC`) but opted to just use the same `SEC` definition (line 81) because they were the same size and represent the same thing. This may come up again with other combined XTCE files.

Reply:
I think you highlight a great observation here. Something to keep in mind for sure. The xtce looks great!